inc/Engine/Media/PreconnectExternalDomains/Frontend/Controller.php

```diff
@@ -206,4 +206,24 @@ private function get_domain_preconnect_item( $domain ) {
 	private function use_prefetch( $domain ) {
 		return wpm_apply_filters_typed( 'boolean', 'rocket_preconnect_external_domains_use_prefetch', false, $domain );
 	}
+
+	/**
+	 * Check if we can let CDN inserts resource hints or not.
+	 *
+	 * @param bool $status Current status.
+	 *
+	 * @return bool
+	 */
+	public function can_cdn_insert_resource_hints( $status ) {
```

Comment:
Let's try to keep up with code standards :sweat_smile: 
```suggestion
	public function can_cdn_insert_resource_hints( $status ): bool {
```